app/views/home/index.html.slim

```diff
@@ -11,20 +11,16 @@
   .govuk-grid-column-one-half
     = m('home.about', headings_start_with: 'xl')
 
-  - unless current_user
-    .govuk-grid-column-one-half
-      .light-grey-box.enrol-box
-        = m('home.login', headings_start_with: 'xl')
-
-        .govuk-button-group
-          = govuk_button_link_to 'Sign in', new_user_session_path
-          .white-space-pre-wrap= ' or '
-          = govuk_link_to 'create an account', new_user_registration_path
-
 .prompt.prompt-home
   .govuk-grid-row
     .govuk-grid-column-one-quarter
       i.fa-2x.fa-solid.fa-circle-info aria-describedby='info icon'
 
     .govuk-grid-column-three-quarters
       = m('home.prompt', headings_start_with: 'xl')
+
```

Comment:
I am not sure of the syntax here but your HTML contains an empty div that is adding spacing. For cleanliness it needs removing - 
![image](https://github.com/DFE-Digital/early-years-foundation-recovery/assets/130461908/e85af608-94c3-42bf-996e-a6b5793dbbd4)


The button at the bottom should be in the grid as follows:
`<div class="govuk-!-margin-top-9 govuk-grid-row"><div class="govuk-grid-column-full"><a data-module="govuk-button" draggable="false" role="button" class="govuk-button govuk-button--start" href="/gov-one/info">Start your training now<svg aria-hidden="true" class="govuk-button__start-icon" focusable="false" height="19" viewBox="0 0 33 40" width="17.5" xmlns="http://www.w3.org/2000/svg"><path d="M0 0h13l20 20-20 20H0l20-20z" fill="currentColor"></path></svg></a></div></div>`

Reply:
Oops, thanks there was a div left over from the old design that I hadn't deleted